### DISCUSSION TITLE
Unable to create a team without providing payment details

### DISCUSSION THREAD

**Discussion by Spyware007** [vercel/community#4169] [Help] [answered]
### Summary

I had 2 teams created which I deleted and now I want to make a new team on a hobby plan.
It is not allowing me to create a new team freely on a hobby plan.
How can I achieve this?

### Steps to Reproduce

Create 2 teams and then delete them.
Then try to create a new team.
This time it will ask for a payment option although I deleted the previously unused teams.

**Comment by amyegan** [comment, marked as answer]
Hi @Spyware007. A Pro plan or trial is required to create a new team. Since you created another team previously, I'm guessing you already used your free [trial](https://vercel.com/docs/accounts/plans/pro/trials#) for the old team which would explain why a payment option is needed to create the new team.

**Comment by Spyware007** [comment]
I am asking, what if I deleted the account and then opened a new account?
Will then I am able to create new teams with hobby plan, right?

**Reply by amyegan** [reply]
Please review our [Fair Use Policy](https://vercel.com/docs/limits/fair-use-policy#) and [Terms of Service](https://vercel.com/legal/terms)
